testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -114,13 +116,13 @@ def test_rms_fit_trj(self):
         # RMSD against the reference frame
         # calculated on Mac OS X x86 with MDA 0.7.2 r689
         # VMD: 6.9378711
-        self._assert_rmsd(fitted, 0, 6.92913674516568)
+        self._assert_rmsd(fitted, 0, 6.929083044751061)
```

Comment:
This change is due to superposition now doing a centering before as well. Basically I'm taking the superposition keyword very serious now.